fcgl/recommendation
# PYTHON TEMPLATE INITIAL IMPLEMENTATION

## Problem
Need a Flask Service up and running to unblock development work

## Solution
Implement initial Flask Service with a health endpoint

## Test  
* Pre-Req: READ THE README and follow set up in this PR

  * **Build:** `docker build -t recommendation .`
  * **Run:** `docker run --name rec -p 80:5000 recommendation`

1. Go to http://0.0.0.0/health/v1/marco
    * Confirm that you get {statusCode: 1, message: 'recommendation'}
2. Go to http://0.0.0.0/
    * Confirm that you see the flask-restplus GUI
